.dockerignore

```diff
@@ -1,3 +1,3 @@
 *
 */*
-!pkg/bin/linux_amd64/consul-ecs
```

Comment:
this line still exists on `main`.. but I don't think that it's required since all the outputs go in to the `dist/` directory. Not an issue here but maybe we should remove this from `main`?

Reply:
I addressed your other comment, here and also tacked on a commit in #118: https://github.com/hashicorp/consul-ecs/pull/118/commits/11b126e7d495645a97639b5a9008ff97d8e15d3c, which will go into main and then will get backported to 0.5.x